extensions/arc/src/localizedConstants.ts

```diff
@@ -341,15 +341,21 @@ export const availableUpgrades = localize('arc.availableUpgrades', "Available Up
 export const version = localize('arc.version', "Version");
 export const releaseDate = localize('arc.releaseDate', "Release Date");
 export const releaseNotes = localize('arc.releaseNotes', "Release Notes");
 export const upgrade = localize('arc.upgrade', "Upgrade");
 export const upgradeDataController = localize('arc.upgradeDataController', "Upgrade Data Controller");
+export const upgradeMiaa = localize('arc.upgradeMiaa', "Upgrade SQL Managed Instance");
 export const areYouSure = localize('arc.areYouSure', "Are you sure you want to apply the selected upgrade?");
-export const upgradeDialog = localize('arc.upgradeDialog', "During a data controller upgrade, portions of the data control plane such as Custom Resource Definitions (CRDs) and containers may be upgraded. An upgrade of the data controller will not cause downtime for the data services (SQL Managed Instance or PostgreSQL server).");
+export const upgradeDialogController = localize('arc.upgradeDialogController', "During a data controller upgrade, portions of the data control plane such as Custom Resource Definitions (CRDs) and containers may be upgraded. An upgrade of the data controller will not cause downtime for the data services (SQL Managed Instance or PostgreSQL server).");
+export const upgradeDialogMiaa = localize('arc.upgradeDialogMiaa', "During a SQL managed instance upgrade, portions of the data control plane such as Custom Resource Definitions (CRDs) and containers may be upgraded. An upgrade of the SQL managed instance will not cause downtime for the data services (SQL Managed Instance or PostgreSQL server).");
 export const monitorUpgrade = localize('arc.monitorUpgrade', "You can check the status of the upgrade by running the following command:");
 export function errorListingLogAnalyticsWorkspaces(error: any): string { return localize('arc.errorListingLogAnalyticsWorkspaces', "Error listing Log Analytics workspaces {0}", getErrorMessage(error, true)); }
 export const noUpgrades = localize('arc.noUpgrades', 'The current version is the latest version. No upgrades available.');
-export const upgrading = localize('arc.upgrading', "Data controller is being upgraded. You can check the status of the upgrade by running the following command: 'kubectl get datacontrollers -A'"); // 'kubectl get datacontrollers -A' should not be localized.
+export const upgradingController = localize('arc.upgradingController', "Data controller is being upgraded. You can check the status of the upgrade by running the following command: 'kubectl get datacontrollers -A'"); // 'kubectl get datacontrollers -A' should not be localized.
+export const upgradingMiaa = localize('arc.upgradingMiaa', "SQL managed instance is being upgraded. You can check the status of the upgrade by running the following command: 'kubectl get sqlmi -A'"); // 'kubectl get sqlmi -A' should not be localized.
```

Comment:
Comments like "**** shouldn't be localized" should be passed as a parameter to the localize function so it gets picked up by the localization team